tmt/utils.py

```diff
@@ -694,7 +694,14 @@ def __init__(self, **kwargs: Any) -> None:
             super().__init__(**kwargs)
 
 
-class Common(_CommonBase):
+class CopyOptionsMeta(type):
+    def __new__(cls: Any, name: str, bases: Tuple[type], dct: Dict[str, Any]) -> Any:
+        x = super().__new__(cls, name, bases, dct)
+        x._options = x._options.copy()
```

Comment:
I'd copy also the `_context`, these two objects are tied together, `_options` are `_context.params`.

Reply:
The metaclass is used during child class inheritance by importlib, and the `_context` parameter is still the default `None` value, so there's nothing to copy.

Reply:
I see, right, there's an initial empty dictionary. Could you add a comment here, that `_context` does not need to be copied? Once we start dealing with `_context` and `_options`, it'd be nice to be reminded of this fact :)